src/Core/GameController.m

```diff
@@ -315,6 +315,7 @@ - (void) beginSplashScreen
 
 - (void) performGameTick:(id)sender
 {
+	[gameView pollControls];
```

Comment:
I think this still needs an #if !OOLITE_SDL guard. We should not have to execute -pollControls: here in the SDL builds.